### PR TITLE
Custom carousel

### PR DESCRIPTION
**Pull request by Fabricevladimir**
Refactored the carousel. Tried to make it fully responsive and mobile friendly

### CODE REVIEW

File: olwebsite-app/src/components/Carousel/Carousel.jsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+      return () => clearInterval(id);
+    }
+  }, [handleNext, interval, autoRun]);
+ 
+  /************************************
+   * Functions
+   ************************************/
+
+  function handlePrevious() {
+    const lastSlide = -SLIDE_SIZE * (slides.length - 1);
```
Comment (ambid17) [inline olwebsite-app/src/components/Carousel/Carousel.jsx]: could move the lastSlide calculation to a utility function as it's used multiple times

File: olwebsite-app/src/components/Carousel/Carousel.jsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+            onMouseEnter={() => setAutoRun(false)}>
+            <FaChevronRight {...arrowStyles}/>
+          </button>
+        </> 
+      }
+      {includeDotIndicators && 
+        <DotIndicator 
+          slides={slides}
+          onDotClicked={index => setPosition(index * -SLIDE_SIZE)}
+          currentIndex={Math.abs(position) / 100}/>
```
Comment (ambid17) [inline olwebsite-app/src/components/Carousel/Carousel.jsx]: you have the magic number "100" here, i think it should be referencing SLIDE_SIZE

Comment (Fabricevladimir) [reply olwebsite-app/src/components/Carousel/Carousel.jsx]: You are correct!

File: olwebsite-app/src/components/Slide/Slide.jsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+export default function Slide ({ image, alt }) {
+  /************************************
+   * Render
+   ************************************/
+
+  return <img src={image} alt={alt} />;
```
[the remark this answers is not in the document]
Comment (Fabricevladimir) [reply olwebsite-app/src/components/Slide/Slide.jsx]: Yeah, I forgot about that. Actually that slide component is unnecessary.

**Pull request merged**